index.html

```diff
@@ -101,6 +102,74 @@ <h2>
         display modes. Otherwise, the [=manifest/display=] member is sufficient
         for most use cases.
       </p>
+
+      <section>
+        <h3>Concepts</h3>
+        <ul>
+          <li>
+            <dfn data-lt="controls of the window">Window controls</dfn>:
+            interface elements that the operating system uses consistently across
+            applications to enable the user to perform certain actions to control
+            the application. Common actions in the [=window controls=] include
+            minimize, maximize/restore, and close buttons.
+          </li>
+          <li>
+            <dfn>Additional Window Controls</dfn>: feature to replace
+            [=window controls=] by calling the minimize/maximize/close etc APIs
+            directly without the need to use browser's native window controls.
+          </li>
+          <li>
+            <dfn data-lt="IWA">Isolated Web App</dfn>: feature for installed
+            and signed web applications which are packaged as Web Bundles and
+            can be used offline instead of being fetched over HTTPS.
+            More: https://github.com/WICG/isolated-web-apps
+          </li>
+          <li>
+            <dfn data-lt="draggable region">Draggable regions</dfn>. To define
+            which regions or elements in the title bar are draggable, this
+            specification defines the CSS <code><dfn>app-region</dfn></code>
```

Comment:
my worry here is that we don't support `app-region` yet, it's currently only supported as `-webkit-app-region`. Should we mention this?

See the note in the web.dev article:
https://web.dev/window-controls-overlay/#determining-which-parts-of-the-title-bar-are-draggable

My guess is that we should mention something like this note? I'm not sure the best wording here.

Reply:
I added a second paragraph regarding the standardization. WDYT?